src/state/endpoints/tests/reducer.test.js

```diff
@@ -0,0 +1,79 @@
+import deepfreeze from 'deepfreeze';
```

Comment:
I see :) I understand, we can still replace it later. eslint --fix should fix the code style :)